packages/dataviews/README.md

```diff
@@ -236,7 +236,8 @@ Array of operations that can be performed upon each record. Each action is an ob
 - `isLoading`: whether the data is loading. `false` by default.
 - `supportedLayouts`: array of layouts supported. By default, all are: `table`, `grid`, `list`.
 - `deferredRendering`: whether the items should be rendered asynchronously. Useful when there's a field that takes a lot of time (e.g.: previews). `false` by default.
-- `onSelectionChange`: callback that returns the selected items. So far, only the `list` view implements this.
+- `onSelectionChange`: callback that signals the user selected one of more items, and takes them as parameter. So far, only the `list` view implements it.
+- `onDetailsChange`: callback that signals the user triggered the details for one of more items, and takes them as paremeter. So far, only the `list` view implements it.
```

Comment:
I think there's a world where this is the same as `onSelectionChange` in other words, when a single item is selected we navigate to "details" and vice versa

Reply:
That's possible, indeed. The way it works now is optimized for being a listing: you navigate through the items, preview them, and can manage them. There's also the multi-selection use case for bulk editing that needs to be accommodated.